exampleFiles/tasks.json

```diff
@@ -6,7 +6,8 @@
         {
             "label": "Run Hollywood Main script",
             "type": "shell",
-            "command": "${config:hollywood.exePath} ${workspaceFolder}/${config:hollywood.mainFile}",
+            "command": "${config:hollywood.exePath}",
+            "args": ["${workspaceFolder}/${config:hollywood.mainFile}"],
```

Comment:
As you mentioned it in #1 could you update your PR and delete the ${workspaceFolder}/ here and on the other locations here in this file? Thank you!